README.md

```diff
@@ -40,6 +40,7 @@ Workflow engine using [WDL](https://github.com/broadinstitute/wdl/blob/wdl2/SPEC
   * [GET /workflows/:version/:id/outputs/:call](#get-workflowsversionidoutputscall)
   * [GET /workflows/:version/:id/logs/:call](#get-workflowsversionidlogscall)
   * [GET /workflows/:version/:id/logs](#get-workflowsversionidlogs)
+  * [GET /workflows/:version/:id/call-metadata](#get-workflowsversionidcallmetadata)
```

Comment:
Agree it's a trivial mod. What I meant was the actual URL might not be right anymore. e.g. `workflow-metadata` might be more apropos